src/Sylius/Bundle/CoreBundle/Resources/config/services/workflow.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <imports>
```

Comment:
```patch
+++ <imports>
+++     <import resource="workflows/*.xml" />
+++     <import resource="workflows/listeners/*.xml" />
+++     <import resource="workflows/callbacks/*.xml" />
+++ </imports>
```
I would vote for wildcards like so.

In long term maintainability, chance to forget to import file or remove import without removing file becomes high, happened to me multiple times, before I've switched to wildcards like in the example.